main.go

```diff
@@ -28,7 +35,38 @@ func main() {
 		cancel()
 	}()
 
-	go http.ListenAndServe(":80", api.New())
+	// Load Environment Variables
+	envConf, err := FromEnv()
+	if err != nil {
+		log.Fatalf("Error getting environment variables %s", err)
```

Comment:
These log messages should use the `go-kit` structured logger.

Reply:
`go-kit` does not seem to have an equivalent of `log.Fatal` so I used the normal `log.Log` function followed by an `os.Exit(1)` when the program fails to initialize its dependencies in `main()`.

Reply:
Ah you're right, go-kit doesn't have that func. We should always be using the structured logger (see the logging section [here](https://github.com/Wattpad/highlander/blob/master/docs/microservice-checklist.md#expected-minimums)), so we can use `logger.Log` + `os.Exit(1)`.